content/information-distribution/signature.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Signature"
+weight: 4
+---
+
+Informo relies on cryptographic signatures in order to verify the integrity of
+data shared across the federation. These signatures are generated using
+asymmetric cryptography keys (`ed25519`, `hmac-sha256`, `hmac-sha512`, etc.),
+which private key solely owned by the end user.
+
+
+## Signed Matrix event
+
+### Event data
+
+Informo signed events are built using a common structure, inspired from [Matrix
+`m.room.encrypted`
+events](https://matrix.org/speculator/spec/HEAD/client_server/unstable.html#m-room-encrypted):
+
+|    Field     |   Type   |                                    Description                                    |
+| ------------ | -------- | --------------------------------------------------------------------------------- |
+| `algorithm`  | `string` | Algorithm used for signing the content. 🔧 Define allowed algorithms               |
+| `sender_key` | `string` | The public part of the key used for signing the event                             |
+| `device_id`  | `string` | 🔧 Optional, ID of the sending device, may be used for Megolm                      |
+| `session_id` | `string` | 🔧 Optional, ID of the session used to encrypt the message, may be used for Megolm |
```

Comment:
I'd rather have :eyes: here, given the description in the doc's about section.

Reply:
I've opened #7 for doing some research about using Matrix E2E keys / Megolm to sign our events. These 🔧 are meant to be removed later with a PR.

Reply:
Roger that, sounds good to me.